src/Runner.Worker/Handlers/OutputManager.cs

```diff
@@ -327,9 +321,11 @@ private DTWebApi.Issue ConvertToIssue(IssueMatch match)
             }
             catch (Exception ex)
             {
-                _executionContext.Debug($"Dropping file value '{match.File}' and fromPath value '{match.FromPath}'. Exception during validation: {ex.ToString()}");
+                _executionContext.Debug($"Dropping file value '{match.File}' and fromPath value '{match.FromPath}'. Exception during validation: {ex}");
             }
 
+            var metadata = new IssueMetadata(null, false, match.SourceText, issueData);
+            var issue = _executionContext.CreateIssue(issueType, match.Message, metadata, true);
```

Comment:
note how `match.SourceText` is stored in the metadata passed to `CreateIssue`.  This is how we achieve log message overrides, now.